packages/web/src/components/collection/desktop/ShareButton.tsx

```diff
@@ -62,4 +65,12 @@ export const ShareButton = (props: ShareButtonProps) => {
       {...other}
     />
   )
+
+  return other.disabled ? (
+    <Tooltip text={messages.emptyPlaylistTooltipText}>
+      <div>{shareButtonElement}</div>
```

Comment:
Let's wrap with span, or for extra credit can we forwardRef the share button element?